### ISSUE TITLE
My OSM PhD is complete!

### ISSUE THREAD

**Issue by edwintse** [OpenSourceMalaria/Series4#75]
My PhD is complete and I'm now a doctor 🍾🎉. Thanks to everyone who's been along for the journey. Next stop, finishing the S4 paper!

My thesis is now online and can be found on The University of Sydney library website [here](https://ses.library.usyd.edu.au/handle/2123/21896). A link has been placed on the [Sources of Data page](https://github.com/OpenSourceMalaria/Series4/wiki/Sources-of-Data) of the wiki under the Theses section as well.

**Comment by MFernflower** [comment]
Your thesis was quite a treat to read! Although a slight thing: Mandrake Fenflower is a nom de plume

**Comment by edwintse** [comment]
@MFernflower Yeah, I know. I opted for that name since that's how we've know you for pretty much the whole time. Hope that's ok :)

**Comment by MFernflower** [comment]
No problems congratulations on the PhD!
